src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/ReasonableAdjustmentsReportITest.java

```diff
@@ -0,0 +1,536 @@
+                                    .add("main_phone", "100000001")
+                                    .add("other_phone", "100000002")
+                                    .add("work_phone", "100000003")
+                                    .add("email", "041500002@email.gov.uk")
+                                )
+                                .add("next_attendance_date", "2024-01-01")
+                                .add("juror_reasonable_adjustment_with_message", new ReportLinkedMap<String, Object>()
+                                    .add("reasonable_adjustment_code_with_description", "M - MULTIPLE")
+                                    .add("juror_reasonable_adjustment_message", "multiple requests")
+                                ),
+                            new ReportLinkedMap<String, Object>()
+                                .add("juror_number", "041500003")
+                                .add("first_name", "CName3")
+                                .add("last_name", "CSurname3")
+                                .add("pool_number_by_jp", "415240101")
+                                .add("contact_details", new ReportLinkedMap<String, Object>()
+                                    .add("main_phone", "200000001")
+                                    .add("other_phone", "200000002")
+                                    .add("work_phone", "200000003")
+                                    .add("email", "041500003@email.gov.uk")
+                                )
+                                .add("next_attendance_date", "2024-01-01")
+                                .add("juror_reasonable_adjustment_with_message", new ReportLinkedMap<String, Object>()
+                                    .add("reasonable_adjustment_code_with_description", "M - MULTIPLE")
+                                    .add("juror_reasonable_adjustment_message", "multiple requests")
+                                ),
+                            new ReportLinkedMap<String, Object>()
+                                .add("juror_number", "041500004")
+                                .add("first_name", "CName4")
+                                .add("last_name", "CSurname4")
+                                .add("pool_number_by_jp", "415240101")
+                                .add("contact_details", new ReportLinkedMap<String, Object>()
+                                    .add("main_phone", "300000001")
+                                    .add("other_phone", "300000002")
+                                    .add("work_phone", "300000003")
+                                    .add("email", "041500004@email.gov.uk")
+                                )
+                                .add("next_attendance_date", "2024-01-01")
+                                .add("juror_reasonable_adjustment_with_message", new ReportLinkedMap<String, Object>()
+                                    .add("reasonable_adjustment_code_with_description", "M - MULTIPLE")
```

Comment:
Maybe worth having at least one other adjustment code other then M to ensure the correct description / title gets pulled